### ISSUE TITLE
Two different items are treated as if they are the same (the icon is the same)

### ISSUE THREAD

**Issue by holymio** [AdiAddons/AdiBags#949]
### Addon Version

v1.10.7

### World of Warcraft Version

Classic

### Describe the bug

I have multiple items with the same icon in my inventory For example these two:
- https://www.wowhead.com/classic/item=22515/shoulderpads-of-faith
- https://www.wowhead.com/classic/item=19370/mantle-of-the-blackwing-cabal

Setting a category for one of the items causes the second items to move to this category aswell. It is not possible to assing them to different categories. I am using the german client of WoW Classic Era.

### Steps to reproduce

1. Get items with the same icon
2. Assign a custom filter for one of the items
3. Assign a different custom filter for the second item
4. Open the bag
5. Check if they are in different categories. For me they are not.

### Verification

- [X] I have disabled all other addons and made sure this bug is triggered only with AdiBags enabled

**Comment by holymio** [comment]
![WoWScrnShot_050123_184424](https://user-images.githubusercontent.com/12158674/235490493-1a507689-1821-411c-b765-f2d17827c7ac.jpg)

Just cleared my WTF folder and tried again, same result.

**Comment by holymio** [comment]
![WoWScrnShot_050923_001908_bug](https://user-images.githubusercontent.com/12158674/236950449-14129136-8748-4dee-bdfb-f91a47d77970.jpg)

So I did a last test to give you feedback regarding this bug. I Changed client language to english, deactivated all addons except of AdiBags, removed AdiBags.lua from my WTF folder. 

For this picture I created a manual filter called A and assigned the Item "Eye of the Dead" to it. Immediately item "Warmth of Forgiveness" moved into the same section.

It is happening with every piece of equipment that has the same icon. But sometimes not immediately, I then first have to equip Item 1, switch to Item 2. Suddenly Item 1 is in the section of Item 2.
